src/ProductBundle/Resources/views/Product/view.html.twig

```diff
@@ -82,6 +82,20 @@ file that was distributed with this source code.
                                     {% endblock %}
                                 </li>
                             {% endblock %}
+                            {% block product_collections %}
```

Comment:
Is this BC? Don't think you want new items in your existing layout when upgrading a minor version of this bundle? Also maybe move it to a dedicated Twig file?